sky/spot/spot_state.py

```diff
@@ -34,9 +37,36 @@
     end_at FLOAT DEFAULT NULL,
     last_recovered_at FLOAT DEFAULT -1,
     recovery_count INTEGER DEFAULT 0,
-    job_duration FLOAT DEFAULT 0)""")
+    job_duration FLOAT DEFAULT 0,
+    failure_reason TEXT,
+    new_job_id INTEGER,
+    task_id INTEGER DEFAULT 0)""")
+_CONN.commit()
 
 db_utils.add_column_to_table(_CURSOR, _CONN, 'spot', 'failure_reason', 'TEXT')
+# Create a new column for job_id, which can be duplicated for sub jobs of the
+# same job.
+# The original job_id will have no actual meaning, but only a unique identifier
+# for all jobs/sub jobs.
```

Comment:
- Update terms in comments?
- "only a unique identifier" --> do you mean "a primary key for the database table"?

Reply:
Updated.

I changed the comment to `The original `job_id`` will no longer have actual meanings, but only a legacy identifier for all tasks in database.` Does this look more clear?